.github/workflows/tests.yml

```diff
@@ -5,11 +5,26 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
```

Comment:
Can you remind me if the hosted environment also runs on ubuntu?

Reply:
Yeah i believe so